N/NOMAD/build_tarballs.jl

```diff
@@ -1,30 +1,40 @@
 using BinaryBuilder, Pkg
 
 name = "NOMAD"
-version = v"4.0.1"
+version = v"4.2.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/amontoison/nomad.git","f6a4b4f18111372e3e7190b019e76ed86b915ddc"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/bbopt/nomad.git", "ad22bef57c84897dab553390ff77c31eaaedf5c3"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd "${WORKSPACE}/srcdir/nomad"
-atomic_patch -p1 "${WORKSPACE}/srcdir/patches/nomad-no-headers-generation.patch"
+mkdir "${WORKSPACE}/path"
+export PATH="${WORKSPACE}/path:$PATH}"
+export WADF=${WORKSPACE}/srcdir/nomad/src/Attribute/WriteAttributeDefinitionFile.cpp
+${CXX_FOR_BUILD} ${WADF} -o ${WORKSPACE}/path/WriteAttributeDefinitionFile
+
+sed s/'set(CMAKE_INSTALL_PREFIX ${PROJECT_BINARY_DIR} CACHE PATH "..." FORCE)'// -i CMakeLists.txt
+
 mkdir build
 cd build
-cmake -DTEST_OPENMP=OFF -DBUILD_INTERFACES=ON -DBUILD_LIBMODE_EXAMPLES=OFF -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+
+cmake -DBUILD_INTERFACE_C=ON \
+    -DTEST_OPENMP=OFF \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release ..
 make -j${nproc}
 make install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = expand_cxxstring_abis(supported_platforms())
-
+platforms = expand_cxxstring_abis(supported_platforms(; exclude=Sys.iswindows))
```

Comment:
I'm not a fan of removing previously working platforms (and windows isn't a niche operating system, sadly).

Reply:
it doesn't compile on mingw nothing i can do about that. i'll add it back whenever https://github.com/bbopt/nomad/pull/82 is closed.